src/applications/coronavirus-screener/containers/App.jsx

```diff
@@ -3,26 +3,30 @@ import MetaTags from 'react-meta-tags';
 import MultiQuestionForm from '../components/MultiQuestionForm';
 import { questions, defaultOptions } from '../config/questions';
 
-export default function App() {
-  return (
-    <div className="covid-screener">
-      <MetaTags>
-        <meta name="robots" content="noindex" />
-      </MetaTags>
-      <div className="vads-l-grid-container">
-        <h1>COVID-19 screening tool</h1>
-        <div className="va-introtext">
-          <p>
-            Please answer the questions listed below. Share your results with
-            the staff member at the facility entrance.
-          </p>
-          <p>We won't store or share your data.</p>
+// eslint-disable-next-line react/prefer-stateless-function
+export default class App extends React.Component {
+  render() {
+    return (
+      <div className="covid-screener">
+        <MetaTags>
+          <meta name="robots" content="noindex" />
+        </MetaTags>
+        <div className="vads-l-grid-container">
+          <h1>COVID-19 screening tool</h1>
+          <h2>Site ID: {this.props.params.id}</h2>
```

Comment:
Not sure if this is just scaffolding, but definitely want to check w/ design + content on adding in this text.

Reply:
oh yeah that was just a debugging placeholder